extensions/mssql/notebooks/TSG/cluster-status.ipynb

```diff
@@ -0,0 +1,159 @@
+            "metadata": {}
+        },
+        {
+            "cell_type": "markdown",
+            "source": "### **Install latest version of mssqlctl**",
+            "metadata": {}
+        },
+        {
+            "cell_type": "code",
+            "source": "import sys, platform\r\n\r\nif platform.system()==\"Windows\":\r\n    user = ' --user'\r\nelse:\r\n    user = ''\r\n\r\ndef executeCommand(cmd, successMsgs, printMsg):\r\n    print(printMsg)\r\n    cmdOutput = !{cmd}\r\n    cmdOutput = ''.join(cmdOutput)\r\n    if any(msg in cmdOutput for msg in successMsgs):\r\n        print(f\"\\nSuccess >> \" + cmd)\r\n    else:\r\n        raise SystemExit(f'\\nFailed during:\\n\\n\\t{cmd}\\n\\nreturned: \\n' + ''.join(cmdOutput) + '.\\n')\r\n\r\ninstallPath = 'https://private-repo.microsoft.com/python/ctp3.1/mssqlctl/requirements.txt'\r\ncmd = f'{sys.executable} -m pip uninstall --yes mssqlctl-cli-storage'\r\ncmdOutput = !{cmd}\r\n\r\ncmd = f'{sys.executable} -m pip uninstall -r {installPath} --yes'\r\nexecuteCommand(cmd, ['is not installed', 'Successfully uninstalled mssqlctl'], 'Uninstalling mssqlctl:')\r\n\r\ncmd = f'{sys.executable} -m pip install -r {installPath}{user} --trusted-host helsinki'\r\ncmdOutput = !{cmd}\r\nexecuteCommand(cmd, ['Requirement already satisfied', 'Successfully installed mssqlctl'], 'Installing the latest version of mssqlctl:')",
+            "metadata": {},
+            "outputs": [],
+            "execution_count": 0
+        },
+        {
+            "cell_type": "markdown",
+            "source": "### **Install latest version of pandas**",
+            "metadata": {}
+        },
+        {
+            "cell_type": "code",
+            "source": "#install pandas\r\ncmd = f'{sys.executable} -m pip show pandas'\r\ncmdOutput = !{cmd}\r\nif len(cmdOutput) > 0 and '0.24' in cmdOutput[1]:\r\n    print('Pandas required version is already installed!')\r\nelse:\r\n    pandasVersion = 'pandas==0.24.2'\r\n    cmd = f'{sys.executable} -m pip install {pandasVersion}'\r\n    cmdOutput = !{cmd}\r\n    print(f'\\nSuccess: Upgraded pandas.')",
+            "metadata": {},
+            "outputs": [],
+            "execution_count": 0
+        },
+        {
+            "cell_type": "markdown",
+            "source": "## **Log in to your big data cluster**\r\nTo view cluster status, you will need to connect to your big data cluster through mssqlctl. \r\n\r\nWhen you run this code cell, you will be prompted for:\r\n- Cluster name\r\n- Controller username\r\n- Controller password\r\n\r\nTo proceed:\r\n- **Click** on the input box\r\n- **Type** the login info\r\n- **Press** enter.\r\n\r\nIf your cluster is missing a configuration file, you will be asked to provide your controller endpoint. (Format: **https://00.00.00.000:00000**)",
```

Comment:
Can we just have format be `ipAddress:port`? The `00` format is confusing here, especially since standard is 3 digits not 2 for most IP parts.